dev/ci/platform-windows.bat

```diff
@@ -0,0 +1,110 @@
+REM @ECHO OFF
+
+REM SET ARCH=64
+REM SET PLATFORM="https://github.com/coq/platform/archive/v8.13.zip"
+REM SET CI_PROJECT_DIR="C:\root"
+
+REM This script builds a minimal Windows platform on Gitlab
+
+ECHO "Start Time"
+TIME /T
+
+REM List currently used cygwin and target folders for debugging / maintenance purposes
+
+ECHO "Currently used cygwin folders"
+DIR C:\ci\cygwin*
+ECHO "Currently used target folders"
+DIR C:\ci\coq*
+ECHO "Root folders"
+DIR C:\
+
+SET CYGROOT=C:\ci\cygwin%ARCH%
+SET DESTCOQ=C:\ci\coq%ARCH%
+SET CYGCACHE=C:\ci\cache\cgwin
+
+CALL :MakeUniqueFolder %CYGROOT% CYGROOT
+CALL :MakeUniqueFolder %DESTCOQ% DESTCOQ
+
+SET CI_PROJECT_DIR_MFMT=%CI_PROJECT_DIR:\=/%
+SET CI_PROJECT_DIR_CFMT=%CI_PROJECT_DIR_MFMT:C:/=/cygdrive/c/%
+SET COQREGTESTING=y
+
+powershell -Command "(New-Object Net.WebClient).DownloadFile('%PLATFORM%', 'platform.zip')"
+REM powershell -Command "Remove-Item '.\platform' -Recurse"
+powershell -Command "Expand-Archive '.\platform.zip' -Force"
+
+cd platform\platform-*
+
+call coq_platform_make_windows.bat ^
+  -arch=%ARCH% ^
+  -destcyg=%CYGROOT% ^
+  -cygcache=%CYGCACHE% ^
+  -cygrepo=https://mirrors.kernel.org/sourceware/cygwin ^

```

Comment:
That's the default and given in the example file only for illustration. I think for CI we should keep the default, so that it changes in CI if the default changes.

Reply:
ok